docs/en/Mounting-Alluxio-FS-with-FUSE.md

```diff
@@ -160,7 +160,10 @@ characteristics, please be aware that:
   shell user group translation service, by setting `alluxio.fuse.user.group.translation.enabled` to `true` 
   in `conf/alluxio-site.properties`. Otherwise `chown` and `chgrp` are no-ops, and `ll` will return the
   user and group of the user who started the alluxio-fuse process. The translation service
-  does not change the actual file permission when running `ll`. 
+  does not change the actual file permission when running `ll`.
+* When `alluxio.fuse.user.group.translation.enabled` is `false`, the user mounted the Alluxio can access all the files and sub-folders.
```

Comment:
is this right, or do they only have access to the files/sub-folders of the user who created the mount?

Reply:
If user `andrew` creates a file `/foo`, then `lu` fuse-mounts Alluxio at `/mnt/alluxio`, can `david` read the content from `/mnt/alluxio/foo`?

Reply:
If  `alluxio.fuse.user.group.translation.enabled` is `false`, the content in `/mnt/alluxio` will have permission `lu:lugroup`, so `lu` can access all the files and subfolders no matter who created the files. 

If  `alluxio.fuse.user.group.translation.enabled` is `true`, the `/foo` file in `mnt/alluxio` will have permission `andrew:andrewgroup`. `lu` and `david` cannot access the file `/foo` if `lu` and `david` do not have the group permission.

The most strange thing here is the user mounted Alluxio-fuse control the permission and access of files, not the Alluxio server.

Reply:
In addition, I mounted an s3 bucket to alluxio `/s3-mnt/` and then fuse-mount Alluxio at `/mnt/alluxio`. The local permission is user:`luuser` and `lugroup`, but the s3 permission is `lu:lu`.
The alluxio folder `/s3-mnt/` has the s3 permission as `lu:lu`.

Now if `alluxio.fuse.user.group.translation.enabled` is `true`, I can fuse-mount the alluxio `/s3-mnt/` folder to local `/mnt/alluxio` but I can not enter the `/mnt/alluxio` because `lu:lu` does not exist in my local computer, the permission check failed.